website/pages/en/sunrise.mdx

```diff
@@ -160,26 +160,26 @@ Once your subgraph has reached adequate curation signal and other Indexers begin
 
 Running infrastructure for your own project is [significantly more resource intensive](/network/benefits/) when compared to using The Graph Network.
 
-Additionally, The Graph Network is significantly more robust, reliable, and cost-efficient than anything provided by a single organization or team. Hundreds of independent Indexers around the world power The Graph Network, ensuring safety, security and redundancy.
+Additionally, The Graph Network is significantly more robust, reliable, and cost-efficient than anything provided by a single organization or team. Hundreds of independent Indexers around the world power The Graph Network, ensuring safety, security, and redundancy.
 
 That being said, if you’re still interested in running a [Graph Node](https://github.com/graphprotocol/graph-node), consider joining The Graph Network [as an Indexer](https://thegraph.com/blog/how-to-become-indexer/) to earn indexing rewards and query fees by serving data on your subgraph and others.
 
 ### Should I use a centralized indexing provider?
 
-If you are building in web3, the moment you use a centralized indexing provider, you are giving them control of your dapp and data. The Graph’s decentralized network offers [superior quality of service](https://thegraph.com/blog/qos-the-graph-network/), reliability with unbeatable uptime thanks to node redundancy, as well as significantly [lower costs](/network/benefits/), and you won’t be held hostage at the data layer.
+If you are building in web3, the moment you use a centralized indexing provider, you are giving them control of your dapp and data. The Graph’s decentralized network offers [superior quality of service](https://thegraph.com/blog/qos-the-graph-network/), reliability with unbeatable uptime thanks to node redundancy, significantly [lower costs](/network/benefits/), and keeps you from being hostage at the data layer.
 
-With The Graph Network, your subgraph is public and anyone can query it openly, which increases the usage and network effects of your dapp. With a centralized indexing solution, the subgraph is private to the centralized provider.
+With The Graph Network, your subgraph is public and anyone can query it openly, which increases the usage and network effects of your dapp. However, with a centralized indexing solution, the subgraph is private to the centralized provider.
 
-Additionally, the upgrade Indexer enables 100,000 free monthly queries for your subgraph before needing to pay for queries.
+Additionally, the upgrade Indexer enables 100,000 free monthly queries for your subgraph before payment is needed for additional queries.
```

Comment:
```suggestion
Additionally, Subgraph Studio provides 100,000 free monthly queries on the Free Plan, before payment is needed for additional usage.
```

Reply:
Should free plan be capitalized? 
Thank you--great catch!